### DISCUSSION TITLE
How to confirm --disable-ikev1 is working.

### DISCUSSION THREAD

**Discussion by vijay-sahu-iitd** [strongswan/strongswan#1974] [Q&A]
Hi, 
To disable  ikev1 , i configured --disable-ike1 flag and compiled the strongswan. 
Now i have two strongswan server : one with ikev1 disabled (using --disable-ikev1) and other without ikev1 disable. Now when i send ikev1 request from client to both both strongswan  server , i see same response from both stronswan server.
How do i know --disable-ikev1 indeed disabled the ikev1 ?

**Comment by tobiasbrunner** [comment]
If you send an IKEv1 Main or Aggressive mode request (you provided no information on what message actually is sent or what the response is, so...) to the host that has IKEv1 disabled, the answer should be an `INVALID_MAJOR_VERSION` notification and a log message saying `received unsupported IKE version 1.0 from ..., sending INVALID_MAJOR_VERSION`.
